homework2.py

```diff
@@ -0,0 +1,4 @@
+m = int(input())
+n = int(input())
+l = int(input())
+print((m + n) * l)
```

Comment:
in this way if my good costs 1 ruble and 50 kopek and I need price for 3 goods it would count (50+1)*3=153 but should count 4 rubles 50 kopeks.